modules/javafx.web/src/main/java/com/sun/webkit/FileSystem.java

```diff
@@ -52,6 +54,19 @@ private static boolean fwkFileExists(String path) {
         return new File(path).exists();
     }
 
+    private static RandomAccessFile fwkOpenFile(String path, String mode) throws FileNotFoundException {
+        return new RandomAccessFile(path, mode);
```

Comment:
It is better to handle the exception and return proper error message/error value to native instead of throwing it. Also log the call and error.